prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = "Johnny Baltazar";
+    console.log(fullName);
+    var numberOfPets = "1";
```

Comment:
For numbers, we want to leave out the quotation marks or else JavaScript will think that we mean a string (text) instead of a numeric value.